src/somef/extract_software_type.py

```diff
@@ -0,0 +1,309 @@
+                                        constants.PROP_TYPE: constants.URL,
+
+                                    },
+                                    1,
+                                    constants.TECHNIQUE_HEURISTICS)
+    elif check_static_websites(path_repo):
+        metadata_result.add_result(constants.TYPE,
+                                    {
+                                        constants.PROP_VALUE: 'https://www.britannica.com/technology/software',
+                                        constants.PROP_TYPE: constants.URL,
+                                        
+                                    },
+                                    1,
+                                    constants.TECHNIQUE_HEURISTICS)
+    elif check_ontologies(path_repo):
+        metadata_result.add_result(constants.TYPE,
+                                    {
+                                        constants.PROP_VALUE: 'https://en.wikipedia.org/wiki/Ontology_(information_science)',
+                                        constants.PROP_TYPE: constants.URL,
+                                        
+                                    },
+                                    1,
+                                    constants.TECHNIQUE_HEURISTICS)
+    elif check_notebooks(path_repo):
+        metadata_result.add_result(constants.TYPE,
+                                    {
+                                        constants.PROP_VALUE: 'https://w3id.org/software-types#Notebook_Application',
+                                        constants.PROP_TYPE: constants.URL,
+                                        
+                                    },
+                                    1,
+                                    constants.TECHNIQUE_HEURISTICS)
+    elif check_workflow(path_repo):
+        metadata_result.add_result(constants.TYPE,
+                                    {
+                                        constants.PROP_VALUE: 'https://biotools.readthedocs.io/en/latest/curators_guide.html?highlight=workflow#tool-type',
+                                        constants.PROP_TYPE: constants.URL,
+                                        
+                                    },
+                                    1,
+                                    constants.TECHNIQUE_HEURISTICS)
+    elif check_command_line(path_repo):
+        metadata_result.add_result(constants.TYPE,
+                                    {
+                                        constants.PROP_VALUE: 'https://w3id.org/software-types#Commandline_Application',
+                                        constants.PROP_TYPE: constants.URL,
+                                        
+                                    },
+                                    0.82,
+                                    constants.TECHNIQUE_HEURISTICS)
+    else:
+        metadata_result.add_result(constants.TYPE,
```

Comment:
So if it's not anything from above, it is software? Contradicting with firs "if"

Reply:
if its not anything of the above it will categorize it as extra, but since i started with putting the links i couldnt make it more clear